docs/releasing.md

```diff
@@ -1,17 +1,25 @@
 # Release process
 
+## Semi-automatic
+
+1. make the release artifacts `make release artifacts`
```

Comment:
```suggestion
1. make the release artifacts `make clean release-artifacts`
```

Reply:
what's the benefit of adding clean? release-artifacts are a PHONY target and will always overwrite whatever exists.

Clean will simply clean up the dev environment a user may already have

Reply:
`clean` will delete existing examples prior to running `release-artifacts` target. This will ensure that the release will have newly generated examples in it.

Reply:
Oh.. I see. the `release-artifacts`doesn't call other make targets. so `clean` not required.